src/prowjobsscraper/utils.py

```diff
@@ -21,3 +22,11 @@ def download_from_gcs_as_string(client: storage.Client, bucket: str, path: str)
     gcs_bucket = client.bucket(bucket)
     gcs_blob = gcs_bucket.blob(path)
     return gcs_blob.download_as_string()
+
+
+def generate_hash(*strings):
+    joined_string = "".join(strings)
+    input_bytes = joined_string.encode("utf-8")
+    hashed_bytes = hashlib.md5(input_bytes).digest()
```

Comment:
Even if we are not in a cryptographic context, I think it would be good to let md5 die, and use a non-cryptographic hash algorithm instead.

Reply:
In case this is changed, I'd suggest to use one that is easily replicated from standard CLI tools (sometimes it's good to be able to calculate the hash on the fly)

Reply:
What do you mean by "easily replicated from standard CLI tools", can you elaborate ?

Reply:
md5 or sha256 have cli util, so that one can easily do this:
```echo 124-step123 | sha256sum | awk '{ print $1 }'
0cf5db53048b0273d797636757d7cf95c7dc51d787d90af3b1ec681359754c54
```

this way we could: check hash for old data easily (without the need of writing up a python script

Reply:
I did not find any library with made CLI tool, but this can be done easy enough like this I believe - 
`python -c "import mmh3; print(mmh3.hash('some string'))"`